WordPressKit/WordPressOrgRestApi.swift

```diff
@@ -40,7 +40,7 @@ public final class WordPressOrgRestApi: NSObject {
     }
 
     enum Site {
-        case dotCom(siteID: UInt64, bearerToken: String)
+        case dotCom(siteID: UInt64, bearerToken: String, apiURL: URL)
         case selfHosted(apiURL: URL, credential: SelfHostedSiteCredential)
```

Comment:
Nitpick: What do you think of making `apiURL` the first associated value, so that it's in line with the  `selfHosted` case?

One argument against doing so would be that `siteID` is more important when it comes to identifying a .com site, so it belongs in first position. I'd be fine with that. This is just a style nitpick / question that I'm not sure on.

Reply:
I'd agree with that argument, considering 'apiURL' is an optional parameter in the initialiser. 😄